app/controllers/api/functions.php

```diff
@@ -945,6 +945,7 @@
             $execution->setAttribute('status', $executionResponse['status']);
             $execution->setAttribute('statusCode', $executionResponse['statusCode']);
             $execution->setAttribute('response', $executionResponse['response']);
+            $execution->setAttribute('stdout', $executionResponse['stdout']);
```

Comment:
Do we want to expose stdout and stderr to the client side API? It might contain sensitive information. What attribute are we using to return logs data?

Reply:
Whether or not to send the logs to client is a decision needed to be made and how to handle it. 
- stdout = logs
- response = users code's response
- stderr = errors

Reply:
@eldadfux renamed but was unable to fix the column even with long column width, column just overflows but the buttons still not align
![image](https://user-images.githubusercontent.com/6360216/184095006-83d4ee0e-ab0e-479b-931b-5e88616b366d.png)